app/manage-projects/[projectSlug]/layout.tsx

```diff
@@ -0,0 +1,229 @@
+
+        {isFinancial ? <TransactionsTrigger /> : null}
+
+        <Tooltip enabled={!canReward} content={<Translate token="common:tooltip.disabledReward" />}>
+          <Button
+            variant={"primary"}
+            size={"sm"}
+            translate={{ token: "manageProjects:detail.activity.actions.reward" }}
+            onClick={() => openRewardFlow({ githubUserIds: [] })}
+            classNames={{
+              base: "max-w-full overflow-hidden",
+              label: "whitespace-nowrap text-ellipsis overflow-hidden",
+            }}
+            isDisabled={!canReward}
+          />
+        </Tooltip>
+      </div>
+    );
+  }, [projectId, isFinancial, canReward]);
+
+  return (
+    <>
+      <PosthogCaptureOnMount
+        eventName={"project_dashboard_viewed"}
+        params={{
+          project_id: projectId,
+        }}
+        paramsReady={Boolean(projectId)}
+      />
+
+      <AnimatedColumn className="h-full">
+        <ScrollView className="flex flex-col gap-md">
+          {openAlert ? <GithubMissingPermissionsAlert onClose={handleCloseAlert} /> : null}
+
+          <RepoIndexingAlert indexingComplete={data?.isIndexingCompleted() ?? true} />
+
+          <PageContent classNames={{ base: "tablet:overflow-hidden" }}>
+            <div className="flex h-full flex-col gap-lg">
+              <header className="flex flex-col flex-wrap items-start justify-between gap-md tablet:flex-row tablet:items-center">
+                <Tabs
+                  variant={"solid"}
+                  searchParams={"data-view"}
+                  tabs={[
+                    {
+                      id: Views.CONTRIBUTORS,
+                      children: <Translate token={"manageProjects:detail.views.contributors"} />,
+                      as: BaseLink,
+                      htmlProps: {
+                        href: NEXT_ROUTER.manageProjects.contributors.root(projectSlug),
+                      },
+                    },
+                    {
+                      id: Views.CONTRIBUTIONS,
+                      children: <Translate token={"manageProjects:detail.views.contributions"} />,
+                      as: BaseLink,
+                      htmlProps: {
+                        href: NEXT_ROUTER.manageProjects.contributions.root(projectSlug),
+                      },
+                    },
+                    {
+                      id: Views.FINANCIAL,
+                      children: <Translate token={"manageProjects:detail.views.financial"} />,
+                      as: BaseLink,
+                      htmlProps: {
+                        href: NEXT_ROUTER.manageProjects.financial.root(projectSlug),
+                      },
+                    },
+                  ]}
+                  selectedId={selectedId}
+                />
+
+                {renderActions()}
+              </header>
+
+              {children}
+            </div>
+          </PageContent>
+        </ScrollView>
+      </AnimatedColumn>
```

Comment:
you can juste return this in the safe ? the panels and posthog can live inside the main component ?